iac/terraform/aws/cognito/cognito.tf

```diff
@@ -57,6 +57,35 @@ resource "aws_cognito_user_pool" "user_pool" {
   }
 }
 
+resource "aws_cognito_user_pool_domain" "set_custom_domain_prod" {
+  count = var.environment == module.environments.PRODUCTION ? 1 : 0
+
+  domain          = "${var.bloodconnect_domain}"
+  user_pool_id    = aws_cognito_user_pool.user_pool.id
+  certificate_arn = var.acm_certificate_arn
+}
+
+resource "aws_cognito_user_pool_domain" "set_custom_domain" {
+  count = var.environment == module.environments.PRODUCTION ? 0 : 1
```

Comment:
maybe just move this into a local block that says

```hcl
locals {
    isProduction = var.environment == module.environments.PRODUCTION ? 0 : 1
}
```

Reply:
Nice idea. Moving it